aya-obj/src/relocation.rs

```diff
@@ -105,7 +110,11 @@ pub(crate) struct Symbol {
 
 impl Object {
     /// Relocates the map references
-    pub fn relocate_maps<'a, I: Iterator<Item = (&'a str, i32, &'a Map)>>(
+    #[cfg(feature = "std")]
+    pub fn relocate_maps<
```

Comment:
This makes aya-obj a lot less useful without std. Do you think it would be too
painful to make this work with both RawFd and BorrowedFd? (Doesn't seem like but
I haven't tried)

Reply:
The previous discussion on this topic is [here](https://github.com/aya-rs/aya/pull/770#discussion_r1310993790). How do you get your hands on an FD in no_std?